ttnn/cpp/ttnn/operations/matmul/device/matmul_op.cpp

```diff
@@ -395,18 +424,42 @@ inline MatmulProgramConfig create_simple_matmul_program_config(
                 (use_mcast_config and mem_config.memory_layout == TensorMemoryLayout::BLOCK_SHARDED)) {
             bool transpose_mcast = input_tensor_a.memory_config().memory_layout == TensorMemoryLayout::BLOCK_SHARDED &&
                                    input_tensor_a.shard_spec().value().orientation == ShardOrientation::COL_MAJOR;
+            uint32_t out_block_h = per_core_M;
+            uint32_t out_block_w = per_core_N;
             out_subblock_h = 4;
             out_subblock_w = 2;
             if (out_subblock_w != per_core_N) {
                 out_subblock_h = 1;
             }
+            if (input_tensor_a.memory_config().memory_layout == TensorMemoryLayout::INTERLEAVED
+                && mem_config.memory_layout == TensorMemoryLayout::INTERLEAVED
+                && input_tensor_b.memory_config().memory_layout == TensorMemoryLayout::INTERLEAVED
+                && input_tensor_a.memory_config().buffer_type == BufferType::DRAM
+                && input_tensor_b.memory_config().buffer_type == BufferType::DRAM
+                && mem_config.buffer_type == BufferType::DRAM
+                && num_cores_x == 8 && num_cores_y == 8) {
```

Comment:
why does this update affects only (8, 8) CoreGrid? In our experience, (8, 8) results in often crashes of device... 
Should we now use (8, 8) by default and it will work? Thanks in advance! 

@dmakoviichuk-tt ^^

Reply:
so this change is for passing the pipelines, otherwise it fails for sweeps. This is just temp fix (mostly for customers to see some good results on WH), we will soon work on a proper fix.

Reply:
I see, thanks for clarification. Can't wait to see improvements in matmul! 

1. do most of our customers use (8, 8) CoreGrid? 
2. did it fail due to performance (time) or crashes issues (max out of L1 cache, etc)?

Reply:
I believe at least one customer is using 8x8 for benchmark MM.
the fail for other grid is the wrong calculation for some params.